DOCS-5309 Amazon MSK Integration Edits + Metrics Versioning Organization (#15070)
DataDog/integrations-core: amazon_msk/README.md

Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.

> ### Prerequisites
> 
> This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][19].

Author: ```suggestion
### Prerequisites

This OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][19].
```

> This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see the [Latest and Legacy Versioning For OpenMetrics-based Integrations][19].

Author: ```suggestion
This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][19].
```

     - <KEY_2>:<VALUE_2>
   ```
   

>    See the [sample amazon_msk.d/conf.yaml][9] for all available configuration options for the latest version. For the legacy version of this integration, see the [legacy example][10].

Author: ```suggestion
   See the [sample amazon_msk.d/conf.yaml][9] for all available configuration options for the latest mode. For the legacy mode of this integration, see the [legacy example][10].
```

[15]: https://docs.datadoghq.com/help/
[16]: https://www.datadoghq.com/blog/monitor-amazon-msk/
[17]: https://docs.datadoghq.com/getting_started/tagging/
[18]: https://docs.datadoghq.com/integrations/amazon_msk

> [19]: https://docs.datadohgq.com/integrations/guide/versions-for-openmetrics-based-integrations

Reviewer: ```suggestion
[19]: https://docs.datadoghq.com/integrations/guide/versions-for-openmetrics-based-integrations
```
